### DISCUSSION TITLE
Help with PseudoBox

### DISCUSSION THREAD

**Discussion by Ionito** [chakra-ui/chakra-ui#2315] [General] [answered]
Hi!
This is my first proyect using chakra, I am building a nextjs typescript app.
The issue I have is when I am trying to use the PseudoBox
I am getting this error:

```
Module '"../../node_modules/@chakra-ui/core/dist/types"' has no exported member 'PseudoBox'.ts(2305)
'PseudoBox' is declared but its value is never read.
```

This is my package.json
`  "dependencies": {
    "@chakra-ui/core": "^1.0.0-rc.3",
    "@chakra-ui/icons": "^1.0.0-rc.3",
    "@sanity/client": "^2.0.1",
    "@sanity/image-url": "^0.140.19",
    "groq": "^2.0.1",
    "next": "latest",
    "react": "^16.13.1",
    "react-dom": "^16.13.1",
    "react-icons": "^3.11.0"
  },
  "devDependencies": {
    "@types/node": "^14.6.0",
    "@types/react": "^16.9.46",
    "@types/react-dom": "^16.9.8",
    "typescript": "4.0.2"
  },`

Any idea of what is happening?
I am getting crazy
Thanks in advance if anyone can help me

**Comment by ljosberinn** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/29307652/97153244-76191d00-1772-11eb-9586-f76e4c6886d7.png)
`PseudoBox` is deprecated in v1 and `Box` should be used instead

**Reply by Ionito** [reply]
Thanks Alex!
